airbyte-integrations/connectors/source-db2/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/Db2SourceDatatypeTest.java

```diff
@@ -312,8 +312,9 @@ protected void initTests() {
             .createTablePatternSql(CREATE_TABLE_SQL)
             .sourceType("TIMESTAMP")
             .airbyteType(JsonSchemaType.STRING)
-            .addInsertValues("null", "'2018-03-22-12.00.00.123'", "'20180322125959'", "'20180101 12:00:59 PM'")
-            .addExpectedValues(null, "2018-03-22T12:00:00Z", "2018-03-22T12:59:59Z", "2018-01-01T12:00:59Z") // milliseconds values are erased
+            .addInsertValues("null", "'2018-03-22-12.00.00.123'", "'2018-03-22-12.00.00.123456'", "'20180322125959'", "'20180101 12:00:59 PM'")
+            .addExpectedValues(null, "2018-03-22T12:00:00.1230Z", "2018-03-22T12:00:00.123456Z", "2018-03-22T12:59:59.0000Z",
+                "2018-01-01T12:00:59.0000Z") // milliseconds values are erased
```

Comment:
this comment is no longer true, right? I.e. millisecond values are now retained?

Reply:
Yes, this comment is no longer relevant. Deleted it.